evalml/tests/automl_tests/test_automl.py

```diff
@@ -1461,6 +1517,33 @@ def test_max_batches_works(mock_pipeline_fit, mock_score, mock_regression_fit, m
     assert automl.full_rankings.shape[0] == n_results
 
 
+def test_early_stopping(caplog, logistic_regression_binary_pipeline_class, X_y_binary):
+    X, y = X_y_binary
+    with pytest.raises(ValueError, match='patience value must be a positive integer.'):
+        AutoMLSearch(X_train=X, y_train=y, problem_type='binary', objective='AUC', max_iterations=5, allowed_model_families=['linear_model'], patience=-1, random_seed=0)
+    with pytest.raises(ValueError, match='tolerance value must be'):
+        AutoMLSearch(X_train=X, y_train=y, problem_type='binary', objective='AUC', max_iterations=5, allowed_model_families=['linear_model'], patience=1, tolerance=1.5, random_seed=0)
```

Comment:
Nit-picking and you probably just moved these but maybe we can separate these first two tests into its own tests, since its more related to invalid values passed in for patience/tolerance?

Reply:
Sure!